src/app/deep-read/requirements.txt

```diff
@@ -9,3 +9,6 @@ flask-restx==0.2.0
 flask-cors==3.0.9
 gdown==3.12.2
 filetype==1.0.7
+fuzzywuzzy==0.18.0
+git+https://github.com/boudinfl/pke.git@6abbcd5d29d14f5151b79a20715ae06dbdd5e4b5
```

Comment:
git?

Reply:
Think we should add spacy to this list too

Reply:
Yes, pke package is available only in git